### DISCUSSION TITLE
conformer_wenetspeech模型优化

### DISCUSSION THREAD

**Discussion by 907122187** [PaddlePaddle/PaddleSpeech#1807] [Q&A] [answered]
感谢大佬提供这么好的环境，在使用中有以下两个问题，麻烦帮忙看看：
1、conformer_wenetspeech模型对部分专业词汇识别效果不佳，有什么方法可以优化？
2、对于部分识别出错的音频，有教程可以对conformer_wenetspeech预训练模型进行二次训练？

**Comment by Jackwaterveg** [comment, marked as answer]
1. 这部分需要后续paddlespeech 支持WFST 的on the fly 功能，从解码器方面进行解决。
2.  目前 wenetspeech 部分的example 还没有建立完整， 可以参考 example/aishell/asr1 的训练过程，对识别出错的音频进行 fine tune。

**Reply by 907122187** [reply]
目前在wenetspeech 识别后，使用[pycorrector](https://github.com/shibing624/pycorrector)对专业词汇进行纠正，在没有“WFST 的on the fly 功能，从解码器方面进行解决”功能之前，可以使用该方法优化专业词汇识别。
